### DISCUSSION TITLE
Request with http POST method / PUT method

### DISCUSSION THREAD

**Discussion by esasakti** [php-webdriver/php-webdriver#814] [Q&A] [answered]
Hi, I want to ask how to use the php webdriver with the http post method and the http put method, right? I saw some examples on the internet using get as the http method ??

**Comment by OndraM** [comment, marked as answer]
Hi, you are probably looking for something like https://github.com/FriendsOfPHP/Goutte .

Php-webdriver acts like user in browser would do, meaning it does only what user can on the page (submit form, click to links etc.)
